scripts/release/tickets.py

```diff
@@ -25,23 +25,28 @@ def get_token():
 
 class Ticket:
     def __init__(self, data):
-        self.id = data["id"]
-        self.fields = {}
-        for field in data["field"]:
-            self.fields[field["name"]] = field["value"]
+        self.id = data["idReadable"]
+        self.fields = dict(
+            (
+                customField["projectCustomField"]["field"]["name"],
+                customField["value"]["name"] if isinstance(customField["value"], dict) else customField["value"],
+            )
+            for customField in data["customFields"]
+        )
+        self.fields["summary"] = data["summary"]
 
     def get(self, field):
         return self.fields[field]
 
     def state(self):
-        return ", ".join(self.get("State"))
+        return self.get("State")
```

Comment:
The new API only returns a single value for each `State` field. I guess multiple `State` fields could be returned but I can't see how it is possible to set this in the UI, and I can't find any issues with more than one value.